server/node/src/main/java/org/kaaproject/kaa/server/admin/services/messaging/MessagingServiceImpl.java

```diff
@@ -202,12 +206,29 @@ public Void call() throws Exception {
     }
   }
 
+  /*
+  * Use instead of UrlParams#generateParamsUrl() cause it uses com.google.gwt.http.client.URL that stop
+  * execution of code on server side.
```

Comment:
The details should be included in the code comment, as many code readers may have the same question.